.gitignore

```diff
@@ -140,3 +140,4 @@ mlruns/
 pytorch\ lightning
 test-reports/
 wandb
+.forked/
```

Comment:
how does this folder happen?

Reply:
Conversion for virtualenvs: .venv for main repo, .forked for personal forked one.